chimney/src/main/scala/io/scalaland/chimney/internal/macros/TransformerMacros.scala

```diff
@@ -587,14 +584,16 @@ trait TransformerMacros extends TransformerConfiguration with MappingMacros with
         Right(resolvedBodyTrees ++ fallbackTransformerBodies)
       } else {
         val errors = unresolvedTargets.toSeq.flatMap { target =>
-          erroredTargets(target) ++ accessorsMapping(target).map { accessor =>
-            MissingTransformer(
-              fieldName = target.name,
-              sourceFieldTypeName = accessor.symbol.resultTypeIn(From).typeSymbol.fullName,
-              targetFieldTypeName = target.tpe.typeSymbol.fullName,
-              sourceTypeName = From.typeSymbol.fullName,
-              targetTypeName = To.typeSymbol.fullName
-            )
+          accessorsMapping(target) match {
+            case Accessor.Resolved(symbol: MethodSymbol, wasRenamed: Boolean) =>
+              MissingTransformer(
+                fieldName = target.name,
+                sourceFieldTypeName = symbol.resultTypeIn(From).typeSymbol.fullName,
+                targetFieldTypeName = target.tpe.typeSymbol.fullName,
+                sourceTypeName = From.typeSymbol.fullName,
+                targetTypeName = To.typeSymbol.fullName
+              ) +: erroredTargets(target)
```

Comment:
The order of reporting errors changed, before we were appending but now we are prepending. I think we could leave the previous behavior (appending) just in case.

Reply:
👍 fixed